tests/.clang-format

```diff
@@ -0,0 +1,104 @@
+#Language:        Cpp
+AccessModifierOffset: -4
```

Comment:
this is nice, does it get applied in the build or test process? if not, how should we enable it?

Reply:
First need install clang-format. Before build process, apply clang-format to committed files (or all files) and save the result into a `tmp` folder. Then compare committed files and files in `tmp` folder. If all files are the same, means submitter do clang-format before send PR.

Current I do it locally to make sure every files have the same style, follow `aclorch.cpp` nearly.
But it shall be another task if we want to enable it in build or test process.

Reply:
I'd like to add enforce the format check on the pr test. can you add clang-format in the sonic-slave-stretch, and add a small script to perform the check. Then, i can run it in the pr test.